examples/autonat/README.md

```diff
@@ -0,0 +1,41 @@
+## Description
+
+This example consists of a client and a server, which demonstrate the usage of the AutoNAT and identify protocols in **libp2p**.
+
+## Usage
+
+### Client
+
+The client-side part of the example showcases the combination of the AutoNAT and identify protocols.
+The identify protocol allows the local peer to determine its external addresses, which are then included in AutoNAT dial-back requests sent to the server.
+
+To run the client example, follow these steps:
+
+1. Start the server by following the instructions provided in the `examples/server` directory.
+
+2. Open a new terminal.
+
+3. Run the following command in the terminal:
+   ```sh
+   cargo run --bin autonat_client -- --server-address <server-addr> --server-peer-id <server-peer-id> --listen-port <port>
+   ```
+   Note: The `--listen-port` parameter is optional and allows you to specify a fixed port at which the local client should listen.
+
+### Server
+
+The server-side example demonstrates a basic AutoNAT server that supports the `/libp2p/autonat/1.0.0` and `/ipfs/0.1.0` protocols.
```

Comment:
`/ipfs/0.1.0` is not a protocol, is this missing the identify bit?

This detailed information is likely to get stale, I'd suggest we only refer to it as "identify" and "autonat".

Reply:
@thomaseizinger Yes you're right, just put "identify" and "autonat" to stay as generic as possible.